src/types.ts

```diff
@@ -225,6 +225,9 @@ export function defaultValue(ctx: Context, field: FieldDescriptorProto): any {
         return 'new Uint8Array()';
       }
     case FieldDescriptorProto_Type.TYPE_MESSAGE:
+      if (isTimestamp(field) && options.useDate === DateOption.STRING) {
```

Comment:
See my comment about the test, I think this is incorrect and can just be removed so it still defaults to undefined.